app/views/team/rename.py

```diff
@@ -5,36 +5,35 @@
 from app import app
 from app.util.auth import get_account
 from app.util.team import rename_team, set_division
+from .team import TeamView
 
+class RenameTeamView(TeamView):
```

Comment:
@manterolat -- It looks like we never actually use the RenameTeamView functionality, and only just use the UpdateTeamView. Unless  you see a spot where we use RenameTeamView, let's just go ahead and remove it